OpenXmlPowerTools/DocumentBuilder.cs

```diff
@@ -2458,8 +3235,8 @@ private static void CopyChartObjects(ChartPart oldChart, ChartPart newChart)
             //{
```

Comment:
Do you need to keep the comments?

Reply:
I just realized these were already here.... but while you're touching the code if these aren't needed they can be removed

Reply:
Removed